src/ragger/navigator/navigation_scenario.py

```diff
@@ -0,0 +1,82 @@
+from abc import ABC
+from pathlib import Path
+from typing import Optional
+
+from .navigator import NavInsID
+
+class NavigationScenarioData:
+    navigation: str
+    validation: str
+    pattern: str
+
+    def __init__(self, device: str, is_address_confirmation: bool, approve: bool):
```

Comment:
I'm not fan of `is_address_confirmation` an `enum` might be more future-proof? Or it could be introduced at a later time as this is not exposed to the apps.

Reply:
Done